python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -485,6 +485,13 @@ class CalibrateImageConfig(pipeBase.PipelineTaskConfig, pipelineConnections=Cali
             "camera model from the obs_* package."
     )
 
+    ap_flux_column = pexConfig.Field(
+        dtype=str,
+        default="base_CircularApertureFlux_12_0_flux",
+        doc="Aperture flux used to generate bg_subtracted_skySource_flux_median "
+        "and bg_subtracted_skySource_flux_stdev."
```

Comment:
I think this description is a bit too specific, and will become out of date if the metrics change. Maybe "used to generate post-subtracted background stats", since that's the phrasing DM-53369 used?